From f15095a372dc44e0daf40fce3ec43cae092006cc Mon Sep 17 00:00:00 2001
From: Tomasz Unger <tomasz.unger@yahoo.pl>
Date: Mon, 23 Feb 2026 20:37:06 +0100
Subject: [PATCH v3 2/2] staging: rtl8723bs: convert single-line comment to
 multi-line format

Convert a long single-line comment to multi-line format
as per coding-style.rst.

Signed-off-by: Tomasz Unger <tomasz.unger@yahoo.pl>
---
v3: Split into two separate patches as requested by Greg Kroah-Hartman.
v2: Added multi-line comment format, changed to codespell.

 drivers/staging/rtl8723bs/core/rtw_wlan_util.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/staging/rtl8723bs/core/rtw_wlan_util.c b/drivers/staging/rtl8723bs/core/rtw_wlan_util.c
index 74c31a97f093..cffb1a9d36fa 100644
--- a/drivers/staging/rtl8723bs/core/rtw_wlan_util.c
+++ b/drivers/staging/rtl8723bs/core/rtw_wlan_util.c
@@ -597,7 +597,9 @@ s16 rtw_camid_alloc(struct adapter *adapter, struct sta_info *sta, u8 kid)
 
 		i = _rtw_camid_search(adapter, addr, kid);
 		if (i >= 0) {
-			/* Fix issue that pairwise and group key have same key id. Pairwise key first, group key can overwrite group only(ex: rekey) */
+			/* Fix issue that pairwise and group key have same key id.
+			 * Pairwise key first, group key can overwrite group only(ex: rekey)
+			 */
 			if (sta || _rtw_camid_is_gk(adapter, i))
 				cam_id = i;
 			else
-- 
2.53.0

